autoPyTorch/api/base_task.py

```diff
@@ -1093,7 +1088,7 @@ def score(
     def __getstate__(self) -> Dict[str, Any]:
         # Cannot serialize a client!
         self._dask_client = None
-        self.logging_server = None
+        self.logging_server = None  # type: ignore [assignment]
```

Comment:
Isn;t it better to define  self.logging_server  as an Optional?

Reply:
in one of the comments you have mentioned that, "BaseContext" has no attribute "Process" motivates to ignore the attr check, therefore I thought to ignore the assignment

Reply:
Oh, ok I see, then never mind. This BaseContext is something we cannot deal with.